jujugui/static/gui/src/app/app.js

```diff
@@ -641,8 +645,10 @@ YUI.add('juju-gui', function(Y) {
 
       // We are now ready to connect the environment and bootstrap the app.
       this.once('ready', function(e) {
-        if (this.get('socket_url') || this.get('sandbox')) {
-          // Connect to the environment.
+        // We only want to connect to the model on application load if we are
+        // in a sandbox or real model and not in gisf.
+        if (!this.get('gisf') &&
```

Comment:
After setting the socket url from the new model in `app.env` you simply need to call `app.env.connect` for it to connect and it'll start to process the incoming deltas as normal.